RELEASE-NOTES.txt

```diff
@@ -15,6 +15,7 @@
 - Fixed an issue where typing a comma in the tag input would insert an empty tag [#1798](https://github.com/Automattic/simplenote-electron/pull/1798)
 - When system theme is selected in Settings, changing the system theme is now immediately reflected in the app [#1801](https://github.com/Automattic/simplenote-electron/pull/1801)
 - Show all checkboxes and search results in note list [#1814](https://github.com/Automattic/simplenote-electron/pull/1814)
+- Fixed markdown render issue with lists with hypehn only on new line [#1822](https://github.com/Automattic/simplenote-electron/pull/1822)
```

Comment:
There's a typo here on `hypehn` but also I feel like the option may fix some other stylistic issues. We might want to borrow from `showdown` itself

```
 - Prevents weird effects in live previews due to incomplete input [#1822](…)
```

Reply:
Done